run

```diff
@@ -486,6 +494,14 @@ copy_nvjpeg_files_() {
     local root_folder=${1:-${TOP}}
     local arch_name="$(get_arch_name_)"
     local nvjpeg_search="${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"
+    if [[ ! -z "${CONDA_PREFIX+x}" ]]
+    then
+      nvjpeg_search="${CONDA_PREFIX:-}/include:${CONDA_PREFIX:-}/lib ${nvjpeg_search}"
+    fi
+    if [[ ! -z "${PREFIX+x}" ]]
+    then
+      nvjpeg_search="${PREFIX:-}/include:${PREFIX:-}/lib ${nvjpeg_search}"
+    fi
```

Comment:
Also change the nvJPEG search paths to included `$PREFIX` & `$CONDA_PREFIX` before the already included paths